### PR TITLE
feat(aws-lambda): Add docs for Python AWS-Lambda Integration 

### PR DESCRIPTION
**Pull request by ahmedetefy**
This PR,
- Updates the Cloud Monitoring table at `product/integrations/` to include both Python (Manual) and Python (Automatic) doc links
- Updates the Implementation Details page at `product/integrations/aws-lambda/how-it-works` to include a How it works section for Python
- Updates both the content and screenshots at `product/integrations/aws-lambda/` to accommodate the changes made for Python Auto instrumentation
- Added a Python doc for AWS-Lambda using Layer at `platforms/python/guides/aws-lambda/layer/`
- Updated `platforms/python/guides/aws-lambda/` to include a link to Python SDK Layer instrumentation doc

### CODE REVIEW

**Comment by scefali** [comment]
@ahmedetefy I think we might need a better way to get users to the manual setup rather than just piling up multiple alerts:

Screen Shot 2021-03-04 at 8 45 35 AM

I suggest combining the content in a single alert with multiple links.

File: src/docs/product/integrations/aws-lambda/how-it-works.mdx [hunk 1]
```diff
@@ -3,17 +3,35 @@ title: Implementation Details
...
+
+   ![](layer_python.png)
+
+  And your Runtime handler will point to Sentry's handler that will initialize the SDK for you, and then call your Runtime handler to invoke your function
+
+   ![](handler_python.png)
+
+* Sentry adds the following environment variables
+  * `SENTRY_INITIAL_HANDLER`: This stores the value of your Runtime handler. If you choose to disable the Sentry layer, this value will be used to set the Runtime handler.
+  * `SENTRY_DSN`: This is set to the DSN of your project
```
Comment (PeloWriter) [inline src/docs/product/integrations/aws-lambda/how-it-works.mdx:34]: Add this link for folks who need to better understand the DSN (I'd also suggest adding the link to line 15)
```suggestion
  * `SENTRY_DSN`: This is set to the [DSN](/product/sentry-basics/dsn-explainer/) of your project
```

File: src/docs/product/integrations/aws-lambda/how-it-works.mdx [hunk 2]
```diff
@@ -3,17 +3,35 @@ title: Implementation Details
...
+  * `SENTRY_TRACES_SAMPLE_RATE`: This sets the sampling rate for transactions. You can manually edit your environment variables if you want a different sampling rate.
```
Comment (PeloWriter) [inline src/docs/product/integrations/aws-lambda/how-it-works.mdx:35]: Suggest adding this link also to line 15
```suggestion
  * `SENTRY_TRACES_SAMPLE_RATE`: This sets the [sampling rate](/platform-redirect/?next=/configuration/sampling/#sampling-transaction-events) for transactions. You can manually edit your environment variables if you want a different sampling rate.
```

File: src/docs/product/integrations/aws-lambda/index.mdx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-title: AWS Lambda (Node)
+title: AWS Lambda (Node/Python)
 sidebar_order: 1
-description: "Learn more about Sentry's AWS Lambda integration and how you can automatically instrument your Node Lambda functions with Sentry without changing your code."
+description: "Learn more about Sentry's AWS Lambda integration and how you can automatically instrument your Node/Python Lambda functions with Sentry without changing your code."
```
Comment (PeloWriter) [inline src/docs/product/integrations/aws-lambda/index.mdx:4]:
```suggestion
description: "Learn more about Sentry's AWS Lambda integration and how you can automatically instrument your Node or Python Lambda functions with Sentry without changing your code."
```

File: src/docs/product/integrations/aws-lambda/index.mdx [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
-title: AWS Lambda (Node)
+title: AWS Lambda (Node/Python)
```
Comment (PeloWriter) [inline src/docs/product/integrations/aws-lambda/index.mdx:2]: We'll end up with a laundry list if we continue this approach.
```suggestion
title: AWS Lambda
```

File: src/docs/product/integrations/aws-lambda/index.mdx [hunk 5]
```diff
@@ -10,7 +10,13 @@ Looking for instructions to set up the Node serverless SDK manually? [Check out
 
 </Alert>
 
-Connect Sentry to your AWS account to automatically add Sentry error and performance monitoring to your Node Lambda functions.
+<Alert title="Note" level="info">
+
+Looking for instructions to set up the Python serverless SDK manually? [Check out these docs instead](/platforms/python/guides/aws-lambda/).
+
+</Alert>
```
Comment (PeloWriter) [inline src/docs/product/integrations/aws-lambda/index.mdx:17]: Let's combine these two, and change their structure. Use the "Note and /Note" approach

File: src/platforms/python/guides/aws-lambda/index.mdx [hunk 6]
```diff
@@ -6,6 +6,13 @@ redirect_from:
 description: "Learn about using Sentry with AWS Lambda."
 ---
 
+<Alert title="Note" level="info">
+
+Looking for instructions on how to add Sentry without modifying your code? [Check out these docs instead](/product/integrations/aws-lambda/).
+
+</Alert>
```
Comment (PeloWriter) [inline src/platforms/python/guides/aws-lambda/index.mdx:13]:
```suggestion
<Note>

Looking for instructions on how to add Sentry without modifying your code? [Check out these docs instead](/product/integrations/aws-lambda/).

</Note>
```

**Review by PeloWriter — APPROVED** [review]

**Comment by PeloWriter** [comment]
I think the page named Implementation Details could use a new name. I'd suggest something along the lines of "Lambda Layer Modifications" or "Changes to Lambda Layers" or similar. Implementation Details, however, isn't terribly useful when viewing in the Navigation.

**Review by scefali — APPROVED** [review]

**Pull request merged**